protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package sfx_log_format;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "gen_go/;log_format";
+
+// model is based off https://github.com/open-telemetry/oteps/blob/master/text/0097-log-data-model.md#log-and-event-record-definition
+message LogRecord {
+    google.protobuf.Timestamp Timestamp = 1;
```

Comment:
google.protobuf.Timestamp is needlessly slow. fixed64 is significantly faster.
I advise for inspiration on fast ProtoBuf schemas to have a look at OTLP trace ProtoBufs which are highly optimized and benchmarked: https://github.com/open-telemetry/opentelemetry-proto/blob/master/opentelemetry/proto/trace/v1/trace.proto